XenAdmin/Dialogs/NewPoolDialog.resx

```diff
@@ -457,7 +457,7 @@
     <value>0</value>
   </data>
   <data name="labelMaster.Text" xml:space="preserve">
-    <value>&amp;Master:</value>
+    <value>&amp;Coordinator:</value>

```

Comment:
Now this has the same hotkey as buttonCreate `&amp;Create Pool`. The hotkeys on the dialog should be unique, or the user will have trouble focusing on the control when clicking Alt+C. Just move the ampersand to a letter not already in use elsewhere in NewPoolDialog.resx.